src/main/java/com/faforever/client/map/generator/GenerateMapTask.java

```diff
@@ -64,7 +64,7 @@ protected Void call() throws Exception {
     ProcessBuilder processBuilder = new ProcessBuilder();
     processBuilder.inheritIO();
     processBuilder.directory(workingDirectory.toFile());
-    processBuilder.command("java", "-jar", generatorExecutableFile.toAbsolutePath().toString(), ".", String.valueOf(seed), version, mapFilename);
+    processBuilder.command("java", "-jar", generatorExecutableFile.toAbsolutePath().toString(), ".", String.valueOf(seed), "0.1.6", mapFilename, "8");
```

Comment:
Is this intentional? You are hardcoding and ignoring the version field. What is the 8 supposed to do?

Reply:
left overs that I did not delete

Reply:
Used to give the gen the number of players via a new arg